assets/admin/students/student-bulk-action-button/index.js

```diff
@@ -12,12 +12,16 @@ import StudentModal from '../student-modal';
 
 /**
  *  Student bulk action button.
+ *
+ * @param {Object}  props
+ * @param {boolean} props.isDisabled Set button's initial state to be disabled or enabled, defaults to disabled.
  */
-export const StudentBulkActionButton = () => {
+export const StudentBulkActionButton = ( { isDisabled = true } = {} ) => {
```

Comment:
What's the purpose of the empty braces here?

Reply:
You're right! Removed them here https://github.com/Automattic/sensei/pull/5056/commits/a1da2256bdb4965a9d08952b35933eb0981c2376 , Thanks! :)